x/concentrated-liquidity/tick.go

```diff
@@ -385,6 +385,15 @@ func (k Keeper) GetTickLiquidityNetInDirection(ctx sdk.Context, poolId uint64, t
 	store := ctx.KVStore(k.storeKey)
 	prefixBz := types.KeyTickPrefixByPoolId(poolId)
 	prefixStore := prefix.NewStore(store, prefixBz)
+
+	// If zero for one, we use reverse iterator. As a result, we need to increment the start tick by 1
+	// so that we include the start tick in the search.
+	//
+	// If one for zero, we use forward iterator. However, our definition of the active range is inclusive
+	// of the lower bound. As a result, current liquidity must already include the lower bound tick
+	// so we skip it.
+	startTick = startTick + 1
```

Comment:
For the follow on PR, would we set the startTick's liquidity to start tick or startTick + 1?

Reply:
For zero for one, we should return current liquidity without including `startTick`'s liquidity net but do return a "liquidity net record" that has it.

For one for zero, we should return current liquidity including `startTick`'s liquidity but no "liquidity net record"